composer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "desandro/imagesloaded",
+  "description": ":love_hotel: Cascading grid layout library http://masonry.desandro.com",
+  "type": "library",
+  "keywords": ["javascript", "library", "grid", "browser", "dom", "layout", "outlayer"],
+  "homepage": "https://github.com/desandro/masonry",
```

Comment:
But the homepage shouldn't be the github repo ? Anyway the description has the webpage.